### ISSUE TITLE
Recreating indexes fails when reloading database fixtures

### ISSUE THREAD

**Issue by phil-davis** [mautic/mautic#5030]
What type of report is this:

| Q  | A
| ---| ---
| Bug report? | Y
| Feature request? | N
| Enhancement? | N

## Description:
The (re)loading of fixtures goes wrong when trying to reset the indexes for lead/company fields.

## If a bug:

| Q   | A
| --- | ---
| Mautic version | current staging cloned from GitHub running in a dev VM 
| PHP version | 5.6.31

### Steps to reproduce:
1. Have the test DB already set up with fixtures...
2. Try to reload the fixtures (e.g. after messing around with the data while testing)
```
php app/console doctrine:fixtures:load --no-interaction --env=test
[2017-09-27 04:16:11]   > purging database
[2017-09-27 04:16:11]   > loading [1] Mautic\UserBundle\DataFixtures\ORM\LoadRoleData
[2017-09-27 04:16:11]   > loading [2] Mautic\UserBundle\DataFixtures\ORM\LoadUserData
[2017-09-27 04:16:13]   > loading [4] Mautic\LeadBundle\DataFixtures\ORM\LoadLeadFieldData

                                                                                                          
  [InvalidArgumentException]                                                                              
  MysqlPlatform::getDropIndexSQL() expects $table parameter to be string or \Doctrine\DBAL\Schema\Table.  
```

**Comment by phil-davis** [comment]
Proposed fix is in PR #5031

**Comment by npracht** [comment]
Hello @escopecz it can be closed :)
